config/environments/production.rb

```diff
@@ -53,14 +55,15 @@
   config.log_level = :debug
 
   # Prepend all log lines with the following tags.
-  config.log_tags = [:request_id]
+  config.log_tags = [ :request_id ]
```

Comment:
Rubocop will bark about this:
```suggestion
  config.log_tags = [:request_id]
```